flink-doris-connector/src/main/java/org/apache/doris/flink/sink/writer/DorisStreamLoad.java

```diff
@@ -269,7 +271,7 @@ private void abortTransaction(long txnID) throws Exception {
         ObjectMapper mapper = new ObjectMapper();
         String loadResult = EntityUtils.toString(response.getEntity());
         Map<String, String> res = mapper.readValue(loadResult, new TypeReference<HashMap<String, String>>(){});
-        if (FAIL.equals(res.get("status"))) {
+        if (FAIL.equals(res.get("status")) || INTERNAL_ERROR.equals(res.get("status"))) {
```

Comment:
Is it better to directly judge **not success**?

Reply:
done